src/app/user-details/user-details.component.scss

```diff
@@ -0,0 +1,44 @@
+* {
```

Comment:
looks like you want to set the font size to `24px`. I would do it in the global styles instead just on this view, so it's consistent across every page

Reply:
and 24px is way too much for a basic font size.

Try

```
html {
font-size: 16px;
}